frontend/src/pages/domain-pack/ui/intent-draft-read-page.module.css

```
@@ -0,0 +1,142 @@
+.pageWrapper {
+  --app-header-height: 72px;
+  display: flex;
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)